<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="room_choice">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1번 방</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2번 방</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3번 방</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4번 방</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5번 방</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6번 방</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7번 방</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8번 방</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9번 방</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10번 방</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>강아지 호텔</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reservation">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>예약 하기</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="start_date_time">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="end_date_time">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>~</string>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>56</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string> ... </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>강아지 이름을 입력해 주세요!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
